dhis2/capture-app
# feat: DHIS2-10759 Implement sections in enrollment form

Added feature for showing custom sections in enrollment form. 
https://jira.dhis2.org/browse/DHIS2-10759

How to test:
1. Go to Maintenance app - Program - choose eg Child Program
2. Go to Attributes - Create Registration Form - add Sections
3. Go to Capture app, register new TEI in Child Program, you should see Sections in the Enrollment form
4. Clear cache
5. Open new event in capture
